rules/falco_rules.yaml

```diff
@@ -2384,7 +2384,7 @@
     not user_known_contact_k8s_api_server_activities
   output: Unexpected connection to K8s API Server from container (command=%proc.cmdline %container.info image=%container.image.repository:%container.image.tag connection=%fd.name)
   priority: NOTICE
-  tags: [network, k8s, container, mitre_discovery]
+  tags: [network, k8s, container, mitre_TA0007_discovery, mitre_TA0006_credential_access, mitre_T1552_unsecured_credentials, mitre_T1552.007_container_api]
```

Comment:
I don't think `mitre_TA0006_credential_access` matches the rule here. Can you elaborate more?

Reply:
Solved